l10n/en/footer-firefox.ftl

```diff
@@ -0,0 +1,47 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# An accessible label used to describe that the role of the element is the primary website navigation.
+footer-landmark-label = Supplementary
+
+footer-download = Download
+footer-download-auto = Download { -brand-name-firefox }
+footer-windows = Windows
+footer-mac = Mac
+footer-ios = iOS
+footer-android = Android
+footer-linux = Linux
+footer-custom-download = Custom Download Options
+footer-latest = Latest Builds
+footer-beta = { -brand-name-beta }
+footer-nightly = { -brand-name-nightly }
+footer-business= { -brand-name-firefox } for Business
```

Comment:
What's the difference between this and the line after? Don't think I've seen "Firefox for Business" ever used before